contents/docs/apps/patterns-connector.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Patterns Connector
+layout: app
+github: https://github.com/PostHog/posthog-patterns-app
+installUrl: https://app.posthog.com/project/apps?name=Patterns
+thumbnail: ../../apps/thumbnails/patterns-logo.svg
+topics:
+    - patterns
+---
+
+### What does the Patterns Connector do?
+
+This Airbyte Export app sends data from PostHog, to Airbyte. It supports both Full Refresh and Incremental syncs. You can choose if this app will copy only the new or updated event data, or all rows in the tables and columns you set up for replication, every time a sync is run.
```

Comment:
Think these accidentally slipped in from the Airbyte app

Reply:
DOH!

```suggestion
The Patterns Connector enables you to send event data from PostHog, to Patterns. This is useful for a number of reasons, including centralizing data into a CDP, or using it to generate customer models. 
```